src/App.js

```diff
@@ -2,21 +2,32 @@ import React, {useState, useEffect} from 'react';
 import TodoList from './TodoList';
 import AddTodoForm from './AddTodoForm';
 
-function useSemiPersistentState() {
+function App() {
   const [todoList, setTodoList] = useState(
     JSON.parse(localStorage.getItem('savedTodoList')) || []
   );
 
+  const [isLoading, setIsLoading] = useState(true);
+
   useEffect(() => {
-    localStorage.setItem('savedTodoList', JSON.stringify(todoList));
-  }, [todoList])
+    const promise1 = new Promise((resolve, reject) => {
```

Comment:
I'm assuming the variable `promise1` is specified by the assignment, but just want to point out that otherwise it'd be better to use a more semantic/specific name

Reply:
Thanks, I updated the name :)